server.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const app = express();
+const apiwrapper = require('./api/api-wrapper');
+const requests = require('./api/requests');
+let oauth2Client;
+apiwrapper.authSetup((authClientResponse) => {
+    oauth2Client = authClientResponse;
+});
+
+// TODO - Change this to use environment variables
+const port = 8081;
+
+const eventTypes = [ 
+    {label : "cop"},
+    {label : "landl"},
+    {label : "learningEvents"} 
+];
+
+// Dynamic endpoint receiving a param of eventtype to test
+app.get('/events/:eventType', function (req, res) {
+    const foundEvent = eventTypes.find((event) => {
+        return event.label == req.params.eventType;
+    });
+    if (foundEvent) {
+        res.send(getData(foundEvent));
+    } else {
+        res.send([]);
+    }
+})
+
+function getData(event) {
+    console.log(oauth2Client);
+    requests.listEvents(oauth2Client);
```

Comment:
This is fine-ish, but it should be async, since `oauth2Client` is returned elsewere, maybe using a promise?